### ISSUE TITLE
[BUG] Refreshed eqlogic appear on summary pages

### ISSUE THREAD

**Issue by KiboOst** [jeedom/core#2933]
4.4.18 Stable

Click on a summary icon to show objects and equipments from this summary.

Staying on the page, all eqlogics that get refreshed (new info etc) are shown, even not being in summary.

**Comment by neuralldev** [comment]
good point, I can also reproduce this. the other eqlogics from the room with refreshed values are progressively appearing at the time they are refreshed.
